### DISCUSSION TITLE
synchronize failed on the 360 browser X

### DISCUSSION THREAD

**Discussion by Kay-26** [floccusaddon/floccus#1120] [Q&A] [answered]
### Which version of floccus are you using?

V4.0.9

### Sync method

{"label"=>"Nextcloud Bookmarks"}

### Describe the Bug

synchronize failed on the 360 browser X 

### Expected Behavior

update bookmark on where other browser，synchronize failed on the 360 browser X 

### To Reproduce

the err msg：
Could not create + #undefined[Bookmarks Bar] parentId: 1, hash: undefined : Bookmark id is invalid. Could not create + #undefined[Other Bookmarks] parentId: 1, hash: undefined : Bookmark id is invalid. Could not create + #undefined[回收站] parentId: 1, hash: undefined : Bookmark id is invalid. Could not create - #undefined[基础运营平台 [prd]](https://plat.winxuan.com/#/dashboard) parentId: 108: Bookmark id is invalid. Could not create - #undefined[协同客户管理运营平台 [prd]](https://dcrm.winxuan.com/#/login) parentId: 108: Bookmark id is invalid. Could not create - #undefined[扩展派-优质chrome插件下载网](https://www.crxfun.com/) parentId: 191: Bookmark id is invalid. Could not create - #undefined[cnBeta.COM - 中文业界资讯站](https://www.cnbeta.com/) parentId: 278: Bookmark id is invalid. Could not create - #undefined[IT之家](https://www.ithome.com/) parentId: 278: Bookmark id is invalid.

![image](https://user-images.githubusercontent.com/69030961/161010133-10d34a73-f6ed-46f9-8de5-ef4ddd7610d4.png)
[Uploading floccus-4.9.0-2022-03-31.log…]()

### Debug log provided

- [X] I have provided a debug log file

**Comment by marcelklehr** [comment]
You may need to set a different local folder to sync to.

**Comment by marcelklehr** [comment, marked as answer]
As such, if it doesn't work with a certain browser, especially more rare ones, it's usually due to that browser rather than to a fault in floccus. Feel free to continue this in discussions, if the problem persists. Once, it's clear that this is a floccus issue, we can move it back to issues.
